### ISSUE TITLE
add dependancies to readme

### ISSUE THREAD

**Issue by tablatronix** [witnessmenow/WiFi-Tetris-Clock#4]

**Comment by witnessmenow** [comment]
I usually prefer to leave them in the [examples](https://github.com/witnessmenow/WiFi-Tetris-Clock/blob/master/ESP32%20or%20TinyPICO/EzTimeTetrisClockESP32/EzTimeTetrisClockESP32.ino#L27), but if people prefer them in the Readme I can add them there too. There is only one dependency common to all examples, Adafruit GFX

**Comment by tablatronix** [comment]
Yeah its usually good practice to put a requirements section in readme
